frontends/web/package.json

```diff
@@ -15,10 +15,10 @@
     "typescript": "^4.4.2"
   },
   "dependencies": {
-    "@zxing/library": "0.18.3",
     "flag-icons": "^6.6.6",
     "i18next": "21.6.16",
     "lightweight-charts": "3.8.0",
+    "qr-scanner": "^1.4.2",
```

Comment:
I'll remove the caret, so we will have to manually upgrade to new version as described here https://github.com/digitalbitbox/bitbox-wallet-app#npm-dependencies